src/makeCalculator.js

```diff
@@ -36,8 +36,32 @@
  *
  * @return {object}
  */
-function makeCalculator() {
-  // write code here
-}
+  function makeCalculator() {
+    let obj = {
+      result: 0,
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/207530888-c9d8ed52-2a99-426e-ab64-b5faa5a8cca2.png)
```suggestion
    const obj = {
      result: 0,
```
Let's use more descriptive names, not just `obj` and `result`.
Also, you don't change the object, so use const instead of let.
Or you can even just immediately return the object